content/geoip2/web-services/using-the-javascript-client.mdx

```diff
@@ -0,0 +1,241 @@
+---
+draft: false
+title: Using the JavaScript Client
+---
+
+Our GeoIP2 JavaScript client allows you to use GeoIP2 Precision services
+client-side without doing any server-side integration on your end.  While it is
+convenient, there are some caveats with its usage as some browser settings and
+add-ons (such as ad blockers) may prevent the GeoIP2 JavaScript API from
+successfully calling the web services, as well as unexpected usage spikes that
+you may want to monitor.
+
+We highly recommend using a server-side integration, as it is more secure and
+more robust than a client-side integration.
+
+## Implementation
+
+Implementing the GeoIP2 JavaScript client involves registering your domain(s),
+adding a JavaScript snippet to your web page, calling the desired API
+method/service, and providing callbacks for all scenarios.
+
+### 1. Register your domain(s)
+
+All domains using the service must be registered. New users may [purchase web
+service lookups](https://www.maxmind.com/en/geoip2-precision-services) and
+receive an account to register domains. Existing account holders are able to
+[register domains directly](https://www.maxmind.com/en/javascript_domains).
+
+### 2. Add a JavaScript snippet to your page
+
+In order to use this service, the following JavaScript must be included in your
+page.
+
+**Do not download this JavaScript file and load it from your server(s)!**
```

Comment:
It's clear to me why this shouldn't be done, but maybe a note to clarify what the consequences are.